modules/ob-creating-a-buildah-build.adoc

```diff
@@ -91,8 +91,9 @@ EOF
 .Example: Using `shp` CLI
 [source,terminal]
 ----
-$ shp build run buildah-golang-build
+$ shp build run buildah-golang-build --follow <1>
 ----
+<1> By using the `--follow` optional flag, you can view the build logs in the output result.
```

Comment:
```suggestion
<1> Optional: By using the `--follow` flag, you can view the build logs in the output result.
```
I am understanding correctly, this flag is optional. If so, this is the correct standard to follow. Please also apply this update to `modules/ob-creating-a-s2i-build.adoc`.

Reply:
Hi @mjpytlak, Thank you for this suggestion. Will it be okay if i cover this suggestion in a new PR for which i have to work. I have recently migrated to Fedora and need some time to set up Git-realted stuff.
This PR is a little bit urgent. Could you please merge it for now.

Reply:
@shipsing Understood. Yes, I will merge this for you. Please address the final comment in a separate PR.

Reply:
Resolving this issue as part of this PR: https://github.com/openshift/openshift-docs/pull/71602